### PR TITLE
UHM-5528: Move encounter types into config

### PR DESCRIPTION
**Pull request by mseaton**

### CODE REVIEW

File: constants.yml [hunk 1]
```diff
@@ -27,19 +27,122 @@ encounterRole:
 
 # for easy lookup, keys should be the name of the encounter type in the DB
```
Comment (mogoodrich) [inline constants.yml:28]: Why the change from camel case to all uppercase with underscore?  To make it more clearly map to the name in the DB?

**Review by mogoodrich — APPROVED** [review]
LGTM!  Curious about the change to all-caps and underscores...

File: constants.yml (hunk 1 above)
Comment (mseaton) [reply constants.yml:28]: These were basically the names of the constant variables in the EncounterTypes.java file, and it was easier to make all of the changes in the code by leaving them alone.  Also, it seemed to align with previous comment that you made about having the plugin add underscores where there was camel case to make the constants more readable - I ended up just changing the constant names to achieve that here.  Thoughts?
